settings.gradle.kts

```diff
@@ -1,13 +1,18 @@
 pluginManagement {
     repositories {
         gradlePluginPortal()
+        maven {
+            url = uri("https://dl.bintray.com/mpetuska/lt.petuska")
+        }
```

Comment:
According to Martynas Petuška (author of `npm-publish`), [the plugin is now on Gradle plugin portal](https://kotlinlang.slack.com/archives/C0B8L3U69/p1603263212387900?thread_ts=1602689960.260900&cid=C0B8L3U69), so this should no longer be needed.

```suggestion
```